cranelift/filetests/filetests/isa/aarch64/call.clif

```diff
@@ -56,24 +52,19 @@ block0(v0: i32):
 
 ; check:  stp fp, lr, [sp, #-16]!
 ; nextln:  mov fp, sp
-; nextln:  sxtw x0, w0
+; check:  sxtw x0, w0
```

Comment:
Just to verify: this becomes a `check` because using the `baldrdash_system_v` adds some instructions we don't care about in this test?

Reply:
More precisely, because it's the only instruction :-)